website/content/docs/server/install.mdx

```diff
@@ -0,0 +1,123 @@
+runner as two separate jobs by default.
+
+~> **NOTE**: **We strongly recommend using Consul** for networking support in Nomad running Waypoint server.
+
+By default, the CLI installer will automatically create a service for Consul in
+Nomad for running Waypoint Server. This is an important default, as it gives
+Waypoint server a stable hostname for your clients to point at, or remote
+runners to use for calling back to the Waypoint server. There are a handful of
+installer flags that let you configure what tags the Waypoint server and UI
+service should have, what datacenter or domain Consul is in, or if you wish you
+may define your own Consul service hostname to use. By default, the
+`waypoint install` helper will return a hostname crafted using the default
+expected host for a Consul service, i.e. `waypoint-server.service.dc1.consul`.
+
+When running the install helper, the CLI should show the Consul DNS backed hostname of Waypoint
+server at the end of an install:
+
+```shell
+waypoint@hashicorp:waypoint λ waypoint install -platform=nomad -accept-tos -nomad-host-volume="waypoint"
+✓ Waypoint server ready
+✓ The CLI has been configured to automatically install a Consul service for
+the Waypoint service backend and ui service in Nomad.
+✓ Server installed and configured!
+✓ Successfully connected to Waypoint server in Nomad!
+✓ Installing runner...
+✓ Waypoint runner installed
+Waypoint server successfully installed and configured!
+
+The CLI has been configured to connect to the server automatically. This
+connection information is saved in the CLI context named "install-1635801894".
+Use the "waypoint context" CLI to manage CLI contexts.
+
+The server has been configured to advertise the following address for
+entrypoint communications. This must be a reachable address for all your
+deployments. If this is incorrect, manually set it using the CLI command
+"waypoint server config-set".
+
+To launch and authenticate into the Web UI, run:
+waypoint ui -authenticate
+
+Advertise Address: waypoint-server.service.dc1.consul:9701
+Web UI Address: https://waypoint-ui.service.dc1.consul:9702
+```
+
+Notice that the advertise and web ui address is shown as a hostname, rather than an IP.
+
+Without using Consul, if the Waypoint server allocation ever gets restarted in Nomad, the IP
+address will no longer be valid, and you will have to upgrade every server clients context.
+There are ways around this issue that are considered more advanced, and are likely
+better candidates for using the [server run](/docs/server/run) flow for setting
+up Waypoint Server.
+
+If you do not wish to automatically register a Consul service in Nomad for
+Waypoint Server by default with `waypoint install`, it can always be disabled
+with the `-nomad-consul-service=false` flag included. You should then see
+an IP address from the Nomad allocation after the Waypoint install helper finishes.
+
+### AWS ECS Platform
+
+This section still needs to be filled out.
```

Comment:
Did you intend to leave this in? I think it's OK if so, just checking.

Reply:
I did. I'm hoping someone after me can pick this up as I'm not super familiar with installing the server to this platform 😄  Maybe @catsby ?

Reply:
This can be covered in a follow-up PR too

Reply:
I think I'll merge this but not backport to the main website. These can go out with the release. Created a new issue we can talk about it tomorrow in our sync: https://github.com/hashicorp/waypoint/issues/2658